includes/main.inc.php

```diff
@@ -221,36 +221,45 @@
 }
 
 // count spam:
-$count_spam_result = mysqli_query($connid, "SELECT COUNT(*) FROM " . $db_settings['forum_table'] . " LEFT JOIN " . $db_settings['akismet_rating_table'] . " ON " . $db_settings['akismet_rating_table'] . ".eid = " . $db_settings['forum_table'] . ".id LEFT JOIN " . $db_settings['b8_rating_table'] . " ON " . $db_settings['b8_rating_table'] . ".eid = " . $db_settings['forum_table'] . ".id WHERE (" . $db_settings['akismet_rating_table'] . ".spam = 1 OR " . $db_settings['b8_rating_table'] . ".spam = 1)");
+$count_spam_sql = 
+	"SELECT COUNT(*) FROM " . $db_settings['forum_table'] . " INNER JOIN (SELECT eid FROM " . $db_settings['akismet_rating_table'] . " WHERE " . $db_settings['akismet_rating_table'] . ".spam = 1 UNION SELECT eid FROM " . $db_settings['b8_rating_table'] . "	WHERE " . $db_settings['b8_rating_table'] . ".spam = 1) spam_list ON spam_list.eid = " . $db_settings['forum_table'] . ".id";
```

Comment:
I know, it's not necessary but the query in entry.inc.php uses the syntax `JOIN (SELECT …) AS foo ON …`, this one not. ***I*** personally prefer the keyword `AS` for better readability. At least we should handle it in a uniform way.

Reply:
I've now updated these two items along with a few other code formatting items I'd come across, to match consistency with existing code.